dev/io.openliberty.grpc.client.1.0.thirdparty/bnd.bnd

```diff
@@ -16,12 +16,13 @@
 
 bVersion=1.0
 
-grpcVersion=1.43.2
+grpcVersion=1.57.2
 
 # Using version=! in order to not have a version attached to the import for packages that were removed
 # from Java after Java 8.  Doing this keeps the import like before Java 11 support. It will get the 
 # packages from Java when using Java 8 or earlier and from the new shipped bundles for Java 9 and later.
 Import-Package: \
+  !com.google.errorprone.*,\
```

Comment:
Same question as previously asked, this is a new package?

Reply:
It is newly used package of Annotations for a google code checking tool. The retention is CLASS which
means that unfortunately they create an OSGi error on startup even though the class loader will not actually
load them up. See above for actual start up message.